test_unstructured_ingest/expected-structured-output/azure/Core-Skills-for-Biomedical-Data-Scientists-2-pages.pdf.json

```diff
@@ -247,7 +323,7 @@
     "text": "Methodology"
   },
   {
-    "type": "Title",
+    "type": "NarrativeText",
```

Comment:
it seems old model performs rather poorly on those category classifications; this one new def makes the category metadata much more accurate and useful (and enables hierarchy)